apps/ui/.env.schema

```
@@ -1 +1 @@
-VITE_APPLICATION_API_URL=
\ No newline at end of file
+VITE_APPLICATION_API_URL=http://localhost:3000
```

always useful to have an empty line at the end of every file

```suggestion
VITE_APPLICATION_API_URL=http://localhost:3000

```